mac/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<title>Schedual</title>
+	<link rel="stylesheet" href="style.css">
+
+</head>
+
+<body>
+
+	<header>
```

Comment:
You should place the header into an html file with just the header for easier copy/paste when templating later